tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -640,36 +722,42 @@ def test_retrieve_object_tags(self, user_attr, expected_status):
                         {
                             "name": "Life on Earth",
                             "taxonomy_id": 1,
-                            "editable": True,
+                            "can_tag_object": True,
                             "tags": [
                                 # Note: based on tree order (Animalia before Fungi), this tag comes first even though it
                                 # starts with "M" and Fungi starts with "F"
                                 {
                                     "value": "Mammalia",
                                     "lineage": ["Eukaryota", "Animalia", "Chordata", "Mammalia"],
+                                    "can_change_objecttag": True,
+                                    "can_delete_objecttag": True,
                                 },
                                 {
                                     "value": "Fungi",
                                     "lineage": ["Eukaryota", "Fungi"],
+                                    "can_change_objecttag": True,
+                                    "can_delete_objecttag": True,
                                 },
                             ]
                         },
                         {
                             "name": "User Authors",
                             "taxonomy_id": 3,
-                            "editable": False,
+                            "can_tag_object": True,
                             "tags": [
                                 {
                                     "value": "test_user_1",
                                     "lineage": ["test_user_1"],
+                                    "can_change_objecttag": True,
```

Comment:
I should have asked this earlier but what does it even mean to change an ObjectTag? Aren't they kind of immutable? (I know we should update `_name` and `_value` if the taxonomy/tag itself has changed, but that's an internal synchronization thing, not a user-triggered request).

I guess for free-text tags it could be possible to edit the value but most UX I've seen for free text tagging makes them immutable in the UI and you just have to delete and re-create the tag to do so.

So I'm wondering if we just drop the `can_change_objecttag`. I noticed it wasn't in this doc for example:

![Screenshot 2024-01-23 at 11 12 12 AM](https://github.com/openedx/openedx-learning/assets/945577/d775a801-af95-4251-aa38-ab3de4f42513)

Reply:
No worries, I've removed `can_change_objecttag`.
One day when we add free-text tags, we may want to resurrect this permission.